### ISSUE TITLE
Health check for Docker container

### ISSUE THREAD

**Issue by Andy-Voigt** [percona/mongodb_exporter#379]
Hello, 

it would be nice to have ### curl installed in this image to have a health check for docker.

Regards, Andy

**Comment by denisok** [comment]
I wonder if we just could add it as option to mongodb_exporter itself. To not add additional binaries/layers to the container we could leverage libs that we already have inside the exporter and just check connection by simple http request, or implement some other way of health checking (rpc connection, signal, whatever).

k8s has external livenessProbe/readinessProbe which looks better and doesn't require additional func.

**Comment by Andy-Voigt** [comment]
The health check in docker must be a command of a container. Any liveniss command with a return of 0 or 1 would be perfect.

**Comment by denisok** [comment]
yep, like `mongod_exporter --isalive` or something like that ) so that would be one binary in a container
